salt/modules/pillar.py

```diff
@@ -62,6 +63,24 @@ def get(key,
             then merging will be skipped and the retrieved value will be
             returned. Earlier releases raised an error in these cases.
 
+    deep_merge_lists : ``None``
```

Comment:
That is a good point. In that case, I think ``merge_nested_lists`` is a better name.